### ISSUE TITLE
Minimal Changes the chart in the auto-update message (Low priority)

### ISSUE THREAD

**Issue by F1reP0wer12345** [MultiMC/Launcher#5158]
### Role

I play multimc with auto update on

### Suggestion

I want the chart to show the issue ID and the NOISSUE/BUG message instead of just if it was an issue or not.

### Benefit

It would make more sense to make the issue ID clickable rather then the description or type.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Basically everything you need to know is described above. It's not a necessity. It just makes more sense. At first I thought it was a bug and it just could not load the issue IDs, and it just displayed NOISSUE instead. But now that I think of it, recently, I did see a bug instead of NOISSUE if I'm not mistaken. But still the issue ID makes more sense. By issue ID i mean #5000 for instance. I took a screenshot of what it looks like now, and how I would do it. Of course I didn't make an application, just grabbed the photo and gave it a quick paint job with gimp. The colors are a little bit off and the underline should be under the ID instead of the NOISSUE, but I'm sure it makes sense. Thanks in advance!
Current Chart
![MultiMC Update Screenshot](https://github.com/MultiMC/Launcher/assets/74012332/32411671-b349-4a24-b7ce-fddf124a4e04)
My Custom Chart:
![MultiMC Update Screenshot - New Layout](https://github.com/MultiMC/Launcher/assets/74012332/710c2061-525a-4afc-8c72-44bb5be80510)

**Comment by phit** [comment]
there is no issue attached to said commits, hence the "NOISSUE", if there was an issue attached it would say GH-5000 already
so I'm kinda lost what you are suggesting

**Comment by F1reP0wer12345** [comment]
Apologies for my late respons. I don't have the free time I used to have. I guess i missed it then. I have only been using MultiMC a little before the 0.7.0 update and never once saw a GH-5000. So I made the suggestion, because I thought it was never there in the first place. Anyway, thanks for sharing with me what the NOISSUE stands for, and thanks for helping with MultiMC's updates/development. I really like the launcher, and couldn't think of going back to the original minecraft launcher. Honestly I don't even know what it looks like. Thanks a lot!
